requirements.txt

```diff
@@ -1,18 +1,17 @@
 addict
-cycler
 descartes
-docutils
 gdspy>=1.5.2
-GeoPandas
+geopandas; platform_system != "Windows"  #pip install geopandas causes fiona errors. Therefore, if you use `conda env`, you must first run `conda install fiona`, while if you use `python venv` you must first download the gdal and fiona wheels from https://www.lfd.uci.edu/~gohlke/pythonlibs/ and manually pip install them in that order.
 ipython
-jedi>=0.17
 matplotlib
 numpy
 pandas
 pint
 pyEPR-quantum>=0.8.4.6
-Pygments
-pyside2<5.15.1
+pygments
+pyside2
 qutip
 scipy
-Shapely
\ No newline at end of file
+shapely  #using conda on Windows OS: pip install of shapely will fail on import for missing geos_c.dll, if <= v1.7.x. For as long as a 1.8.0 version is not released (already patched in github), you will need to run `conda install shapely` (before installing qiskit_metal).
```

Comment:
I am still confused about the "using conda on Windows OS" part. If you `pip install shapely` with the conda Python, then you get this error, but if you `pip install shapely` with a different installation of Python, then you don't get the error?

Reply:
that is correct.
the shapely in pip only behaves weird in conda. And also not consistently. I documented this in the FAQ as well for those that do not see this file (like PyPI users)